cornflow-dags/DAG/run_deployed_dags.py

```diff
@@ -0,0 +1,126 @@
+import json
+import logging
+import time
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.python import PythonOperator
+from airflow.secrets.environment_variables import EnvironmentVariablesBackend
+from airflow.utils.db import create_session
+from cornflow_client import CornFlowApiError
+from cornflow_client.airflow.dag_utilities import connect_to_cornflow
+
+default_args = {
+    "owner": "baobab",
+    "depends_on_past": False,
+    "start_date": datetime(2020, 2, 1),
+    "email": [""],
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": -1,
+    "retry_delay": timedelta(minutes=1),
+    "catchup": False,
+}
+
+logger = logging.getLogger("airflow.task")
+
+
+def run_examples(**kwargs):
+    with create_session() as session:
+        current_examples = {
+            var.key: json.loads(var.get_val())["instance_1"]
+            for var in session.query(Variable)
+            if "_examples" in var.key
+        }
+
+    current_examples = {k: v for k, v in current_examples.items() if v != {}}
+
+    cf_client = connect_to_cornflow(EnvironmentVariablesBackend())
+    executions = []
+
+    for key, instance in current_examples.items():
+        schema = key.split("z_")[1].split("_examples")[0]
+
+        try:
+            response = cf_client.create_instance(
+                data=instance, name=f"Automatic_instance_run_{schema}", schema=schema
+            )
+        except CornFlowApiError as e:
+            logger.info(e)
+            logger.info(
+                f"Instance example for schema {schema} had an error on creation"
+            )
+            continue
+
+        instance_id = response["id"]
+
+        config = {"timeLimit": 60, "msg": True, "seconds": 60}
```

Comment:
@marioncottard I am getting a lot of weird behaviours with this part of timeLimit and seconds. Could you start preparing a PR that modifies all example DAGs to use the translation dictionaries that we had created?

Reply:
Okay